api/v1/rpc_bootstrap_cluster.go

```diff
@@ -7,10 +7,11 @@ const BootstrapClusterRPC = "k8sd/cluster"
 
 // BootstrapClusterRequest is the request message for the BootstrapCluster RPC.
 type BootstrapClusterRequest struct {
-	Name    string          `json:"name"`
-	Address string          `json:"address"`
-	Config  BootstrapConfig `json:"config"`
-	Timeout time.Duration   `json:"timeout"`
+	Name              string          `json:"name"`
+	Address           string          `json:"address"`
+	Config            BootstrapConfig `json:"config"`
+	Timeout           time.Duration   `json:"timeout"`
+	ContainerdBaseDir string          `json:"containerdBaseDir"`
```

Comment:
This should be part of the Bootstrap/Join config not the request
https://github.com/canonical/k8s-snap-api/blob/main/api/v1/type_bootstrap_config.go#L6

Reply:
Done